articles/azure/azs-how-setup-recovery-portal.md

```diff
@@ -0,0 +1,486 @@
+
+    * Volume Shadow Copy Service (VSS) ensures that apps on the VM are in a consistent state when the snapshot is taken.
+
+6. Select **OK** to create the policy.
+
+    ![Azure site recovery infrastructure 2](images/azs-browser-recovery-infrastructure-configuration2.png)
+
+### Adding admin account
+
+After the setup has complete if you navigate to the configuration servers desktop you will find a tool called **Cspconfigtool**, run this tool and navigate to the **Manage Accounts** tab, select **Add Account** and then fill out the details matching the admin accounts details on the VM you wish to replicate. It may take a while for the configuration server to sync with Public Azure
+
+![Azure site recovery account manager](images/azs-siterecovery-account-manager.png)
+
+### Enable replication
+
+Make sure you've completed all the tasks in **Step 1: Preparation of VMs**. On Public Azure then enable replication as follows:
+
+1. Under **Protected items** in the vault select **Replicated items**.
+
+2. Select **Replicate**.
+
+3. In Source, select **On-premises**.
+
+4. In **Source location** select the configuration server you set up.
+
+5. In Machine type, select Physical machines.
+
+6. Select the process server (configuration server).
+
+    ![Azure replication source](images/azs-browser-replication-source.png)
+
+7. In Target, select the **subscription** and the **Post-failover resource group** in which you want to create the VMs after failover. Choose the **Post-failover deployment model** that you want to use for the failed-over VMs.
+
+8. Select the Public Azure storage account in which you want to store replicated data or create a new storage account.
+
+9. Select **Configure now** for selected machines to apply the network setting to all machines you select for protection. Select **Configure later** if you want to select the Azure network separately for each machine.
+
+10. Select or create a new Azure network and subnet to which Azure VMs connect when they're created after failover.
+
+    ![Azure replication target](images/azs-browser-replication-target.png)
+
+11. In Physical Machines, select **Physical machine**. Specify the name, IP address and the operating system of each machine you want to replicate, you should have noted these down from when you set up the machine prerequisites.
+
+    * Use the internal IP address of the machine.
+
+    **If you specify the public IP address replication might not work.**
+
+    ![Azure replication machine](images/azs-browser-replication-machine.png)
+
+12. In Properties > Configure properties, select the account that the process server will use to automatically install Mobility Service on the machine. If the configuration server has yet to sync with Public Azure, no account will appear here.
+
+    ![Azure replication properties](images/azs-browser-replication-properties.png)
+
+13. In Replication settings > Configure replication settings, check that the correct replication policy is selected.
```

Comment:
Select the Replication settings step and in the *Configure replication settings* blade, check that...

Reply:
done